public/employers/virgin-atlantic.yml

```diff
@@ -0,0 +1,34 @@
+name: Virgin Atlantic Airways Limited
+aliases:
+  - Virgin Atlantic International Limited
+shortName: Virgin Atlantic
+wiki: Virgin Atlantic
+image: virgin-atlantic.svg
+officialWebsite: https://corporate.virginatlantic.com/gb/en/our-story.html
+industries:
+  - airline
+location:
+  city: Crawley
+  country: gb
+  international: true
+  wiki: Crawley
+employeesBefore:
+  type: exactly
+  upperBound: 8571
+  year: 2018
+summary: Even though it made executive pay cuts, Virgin Atlantic laid off around a third of its staff two months later.
+citations:
+  - summary: Virgin Atlantic cut its executives' pay by upwards of 15%.
+    positivity: 1
+    type: publication
+    sources:
+      - source: MarketWatch
+        link: https://web.archive.org/web/20200409155919/https://www.marketwatch.com/story/virgin-atlantic-cuts-ceo-pay-as-airline-industry-predicts-113-billion-hit-from-coronavirus-2020-03-05
+        date: 2020-03-06T10:50:00-05:00
```

Comment:
Wrong date, should be 03/05.

Reply:
I... dunno. The article header says *Published: March 6, 2020 at 10:50 a.m. ET*, and the publication tag is `<meta name="parsely-pub-date" content="2020-03-06T10:50:00-05:00">`. The URL says 3/5 but far as I can tell, it's 3/6.